.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "template-component-package",
+  "containerUser": "ros2",
+  "initializeCommand": [
+    "docker",
+    "build",
+    "--file=aica-package.toml",
+    "--target=development",
+    "--build-arg=config.signing.enabled=false",
+    "--tag=aica-technology/template-component-package:development",
+    "."
+  ],
+  "image": "aica-technology/template-component-package:development",
+  "postStartCommand": "rm -r /home/ros2/ws/src/template_component_package && ln -s ${containerWorkspaceFolder}/source/template_component_package /home/ros2/ws/src/template_component_package",
```

Comment:
Instead of setting
```
"workspaceMount":  "source=${localWorkspaceFolder},target=/home/ros2/.devcontainer,type=bind,consistency=cached",
"workspaceFolder": "/home/ros2/.devcontainer",
```
which don't have an effect in the remote container, I now propose to use `containerWorkspaceFolder` which seems to work both in the local and remote devcontainer. At this point, I can't even remember why we use the `.devcontainer` directory everywhere. Not adding `workspaceFolder` and `workspaceMount` the default directory is just `/workspace/<repo-name>` which seems equally fine to `~/.devcontainer`

Reply:
https://code.visualstudio.com/remote/advancedcontainers/change-default-source-mount